android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -128,49 +128,15 @@
       return true;
     }
 
-    function setKeymanLanguage(keyboardName, internalName, languageName, langId, kbdFile, font, oskFont, package) {
-      var kmw=window['keyman'];
-      var kbdInterface=window.KeymanWeb;
-
-      // Defaults for missing arguments
-      switch (arguments.length) {
-        case 1:
-          internalName = keyboardName;
-          if (keyboardName == 'English') {
-              internalName = 'us';
-          }
-          break;
-        case 2:
-          languageName = keyboardName;
-          break;
-        case 3:
-          langId='';
-          break;
-        case 6:
-          oskFont = font;
-          break;
-      }
-
-      var k = {
-        KN:keyboardName,
-        KI:'Keyboard_'+internalName,
-        KLC:langId,
-        KL:languageName,
-        KF:kbdFile,
-        KP:package
-      };
-      if (font) {
-        k.KFont = font;
-        //window.console.log('keyboard.html assigning k.KFont = ' + font);
-      }
-      if (oskFont) {
-        k.KOskFont = oskFont;
-        //window.console.log('keyboard.html assigning k.KOskFont = ' + oskFont);
-      }
-      kbdInterface.registerStub(k);
+    function setKeymanLanguage(k) {
+      KeymanWeb.registerStub(k);
+      keyman.setActiveKeyboard(k.KP + '::'+k.KI, k.KLC);
```

Comment:
While working on improved globe key button, I noticed some weirdness in switching keyboards.
I think cause `KeymanWeb.registerStub(k)` is internally changing k.KI to be namespaced

https://github.com/keymanapp/keyman/blob/651932f1bbbb6d7f724b2e602e0c042f227bd982/web/source/kmwembedded.ts#L165

Not sure why it's changed now...

Reply:
@MakaraSok I think this line is related to the screenshots you posted yesterday.

```suggestion
      keyman.setActiveKeyboard(k.KI, k.KLC);
```

Reply:
Ouch, now I understand this. It would be far better for registerStub to not be mutating this parameter (deep copy first)! @jahorton, what do you think?

Reply:
I'm cool with that.  So, sounds like we'll throw #5438's fix (when ready) in with this PR once that one's ready?  Or will we keep those bits separate?  (Either way, sounds like we're rejecting the code suggestion above as-is.)

Reply:
> So, sounds like we'll throw #5438's fix (when ready) in with this PR once that one's ready?

Yes. I think that makes sense. The issue that it is fixing is introduced in stable-14.0 in this PR, so happy to hold off until that one is approved and tested and then cherry-pick it into this PR.

Reply:
#5438 is approved and merged, so it can be 🍒 picked to this PR now